### ISSUE TITLE
Unsent messages disappear.

### ISSUE THREAD

**Issue by vladbortnik** [BoltAI/BoltAI#123]
An unsent message would disappear if I need to switch to a previous chat even for 1 second. Please fix.

**Comment by longseespace** [comment]
Confirmed this is a regression bug. A fix will be release soon.

**Comment by brandondrew** [comment]
Losing user data is one of the worst things you can do in any app.

**Comment by longseespace** [comment]
> Losing user data is one of the worst things you can do in any app.

I agree. TBF this is unsent message, not messages already in a chat. But I'm working on the fix rn

**Comment by longseespace** [comment]
I've fixed this in the latest version v1.17.1. Can you confirm? Sorry again for the issue.

**Comment by latent-capital** [comment]
this is remarkable, because in the present implementation users might wish the messages **would** disappear, instead of their viewport being truncated irreversibly #185
